packages/prompts/src/index.ts

```diff
@@ -509,9 +510,9 @@ export const groupMultiselect = <Value>(opts: GroupMultiSelectOptions<Value>) =>
 			return `${color.strikethrough(color.dim(label))}`;
 		}
 		if (state === 'active-selected') {
-			return `${color.dim(prefix)}${color.green(S_CHECKBOX_SELECTED)} ${label} ${
-				option.hint ? color.dim(`(${option.hint})`) : ''
-			}`;
+			return `${color.dim(prefix)}${color.green(
```

Comment:
do you know what reformatted this?

if you run `pnpm format`, does it put it back?

Reply:
biome did i believe?

Reply:
strange that in main it is the same code but formatted it differently 🤔

Reply:
let me try to rerun the cmd, i was dealing with errands 🫣